src/sql/platform/connection/test/common/connectionConfig.test.ts

```diff
@@ -384,6 +398,18 @@ suite('ConnectionConfig', () => {
 		});
 	});
 
+	test('getConnections should return connections sorted alphabetically by title given datasource.connectionSort is set to \'byTitleAlphabetically\'', () => {
```

Comment:
Do we already have a test for the sorting by date?

Reply:
With current implementation, connections are by default sorted by date, so the existing test case should be sufficient for that.

Reply:
added test cases for sorting by date